test/migration/verify/verify_migration_test.go

```diff
@@ -110,20 +110,26 @@ func verifySecondMemberProvisionedSpace(t *testing.T, awaitilities wait.Awaitili
 
 func verifyProvisionedSignup(t *testing.T, awaitilities wait.Awaitilities, signup *toolchainv1alpha1.UserSignup) {
 	cleanup.AddCleanTasks(awaitilities.Host(), signup)
-	VerifyResourcesProvisionedForSignup(t, awaitilities, signup, "base", "base")
+	// TODO temporarily check only Spaces until UserTier migration is completed
+	VerifyResourcesProvisionedForSignup(t, awaitilities, signup, "deactivate30", "base")
+	// VerifySpaceRelatedResources(t, awaitilities, signup, "base")
```

Comment:
I'm confused - the newly added line that is commented out `VerifySpaceRelatedResources` is part of the `VerifyResourcesProvisionedForSignup` anyway

Reply:
That was something I was trying when I initially started writing the tests. At the time I was trying to make only new users use UserTiers but then I realized it was unnecessarily complicating things and it was easier to migrate all existing users to use UserTiers and make new users use UserTiers as well. So that commented out code is obsolete. Sorry for the confusion.